.github/workflows/shared-build-deploy.yaml

```diff
@@ -7,14 +7,15 @@ name: Build and Deploy OHQ
 on: push
 
 jobs:
-  # backend-check:
-  #   name: "Backend Check"
-  #   uses: pennlabs/shared-actions/.github/workflows/django-check.yaml@v0.1
-  #   with:
-  #     projectName: officehoursqueue
-  #     path: backend
-  #     flake: true
-  #     black: true
+  backend-check:
+    name: "Backend Check"
```

Comment:
This is penn-mobiles yaml file:
```yaml
  backend-check:
    name: "Backend Check"
    uses: pennlabs/shared-actions/.github/workflows/django.yaml@507f4cf1cc88259fc8625fb3a35bd3f18fc91cd5
    secrets: inherit
    with:
      projectName: pennmobile
      path: backend
      pythonVersion: 3.11-bookworm
      flake: true
      black: true
      ruff: false
```

Reply:
Would suggest adding the `secrets: inherit` and adding a `ruff: false`

Reply:
done